tmp.js

```diff
@@ -8,6 +8,7 @@ if(userProjects==null || userProjects.size()<1){
     <thead>
         <tr>
```

Comment:
this file appears to be leftover and should not be here!
you can leave it untouched -- or even do `git rm tmp.js` if you wish.